src/components/AvatarWithDisplayName.js

```diff
@@ -69,15 +70,20 @@ const AvatarWithDisplayName = (props) => {
                             mainTooltip={props.report.ownerEmail}
                             secondaryTooltip={subtitle}
                             size={props.size}
+                            noMargin
```

Comment:
This change doesn't look related to this PR.

Reply:
I don't think we want this change either. If you think it's a bug than it's better to report in slack first. Let's try to keep this PR limited to what actually we are trying to handle.